### ISSUE TITLE
Multi-task (multi-classification + regression)

### ISSUE THREAD

**Issue by ChihYuLee** [GAMES-UChile/mogptk#76]
Hi, 

I am wondering if there is a way we can combine multi-class classification with regression. I probably need to write a CategoricalLikelihood().
My input features are 2 dimensional output is 1D regression + 1D 3-classes categories. 
My current code is: 

----------------------------------------------
np.random.seed(42)
n_samples = 100
X1 = np.random.rand(n_samples) * 10  
X2 = np.random.rand(n_samples) * 10 

y_regression = 3 * X1 + 2 * X2 + np.random.randn(n_samples) * 2  
y_classification = np.digitize(y_regression, bins=[-np.inf, 10, 25, np.inf]) - 1 
y_classification = y_classification.astype(int)
y_regression = np.array([2.5 if label == 0 else 0.5 if label == 1 else 1.5 for label in y_classification])+np.random.randn(n_samples) * 2 *0.1

plt.figure(figsize=(4, 3))
plt.scatter(X1,X2, c=y_regression)
plt.title('Regression')
plt.colorbar()

plt.figure(figsize=(4, 3))
plt.scatter(X1,X2, c=y_classification)
plt.title('Classification')

X = np.hstack([X1[:,None], X2[:,None]])
y = np.hstack([y_regression[:,None], y_classification[:,None]])
X_train = torch.from_numpy(X).float()
y_train=torch.tensor(y)

num_latents = X_train.shape[1]
num_tasks = y_train.shape[1]
num_classes= 3

# Choose likelihood
likelihood = mogptk.gpr.MultiOutputLikelihood(
    mogptk.gpr.BernoulliLikelihood(), 
    CategoricalLikelihood())

# Choose inference
inference = mogptk.Hensman(likelihood=likelihood)

# Choose kernel
kernel = mogptk.gpr.LinearModelOfCoregionalizationKernel(
    mogptk.gpr.SquaredExponentialKernel(),
    mogptk.gpr.SquaredExponentialKernel(),
    output_dims=num_tasks)

# Set up model
model = mogptk.Model(data, kernel, inference=inference)

I would appreciate your insights and advice.

**Comment by tdewolff** [comment]
Hi ChihYuLee, yes the categorical likelihood, or softmax, is missing form the likelihoods. You can have a go at implementing it if you want, we'd be happy to accept a PR!
